app/api/v1/endpoints/sso.py

```
@@ -103,9 +104,31 @@ def _complete_sso_login(
         db.commit()
+        if clear_forced_password_change:
+            log_security_event(
+                logger,
+                "sso_forced_password_change_cleared",
+                req,
+                "Cleared unsatisfiable forced password change for SSO-only user",
+                user_id=sso_user.id,
+                username=sso_user.username,
+                auth_method=sso_user.auth_method,
+            )
     except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)